### PR TITLE
Feature/custom-date

### PR DESCRIPTION
**Pull request by saharj**
# Description
Added a datePicker component for developers to be able to test the app for any custom date and time

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/MatchUp/FaceoffContent.js [hunk 1]
```diff
@@ -50,7 +52,7 @@ const FaceoffSubDisplay = ({ sub, type, feedback, ...props }) => {
   const [showModal, setShowModal] = useState(false);
   const [locked, setLocked] = useState(true);
 
-  const currentDate = new Date();
+  const currentDate = props.custom_date ? props.custom_date : new Date();
```
Comment (Jahteo) [inline src/components/pages/MatchUp/FaceoffContent.js:55]: I like how this pulls the custom date only if that prop exists. Nice.

File: src/components/pages/MatchUp/MatchUpContainer.js [hunk 2]
```diff
@@ -102,6 +102,7 @@ export default connect(
     child: state.child,
     faceoffs: state.faceoffs,
     votes: state.votes,
+    custom_date: state.date.custom_date,
```
Comment (Jahteo) [inline src/components/pages/MatchUp/MatchUpContainer.js:105]: Is this file actually using the custom_date?

**Review by Jahteo — APPROVED** [review]
Overall looks good. I support merging it.

File: src/components/pages/MatchUp/MatchUpContainer.js (hunk 2 above)
Comment (saharj) [reply src/components/pages/MatchUp/MatchUpContainer.js:105]: Yes, this is the parent component for `RenderMatchUp` and `FaceoffContent`. By adding `custom_date` to the parent component, the data will be passed to its children as props, so we won't need to connect each child to the Redux. The main reason I went with this approach is that we get the same result with the minimum amount of code.

**Review by VictorDronov — APPROVED** [review]
Nice Job. This gives us the ability to chose how far along faceoffs are?

**Pull request merged**